mithril-aggregator/src/command_args.rs

```diff
@@ -244,7 +244,7 @@ impl ServeCommand {
         let _ = shutdown_tx.send(());
 
         info!("Event store is finishing...");
-        event_store_thread.await?;
+        event_store_thread.await.unwrap();
```

Comment:
Is there any specific reason why you unwrapped here?

Reply:
yes, it should not change anything since the event store returns `Ok(())` I wanted to make it to fail as fast as possible in case of … whatever.